app/src/main/java/org/vss/impl/postgres/sql/v0_create_vss_db.sql

```diff
@@ -1,8 +1,7 @@
 CREATE TABLE vss_db (
-    store_id character varying(120)  NOT NULL,
-    key character varying(120)  NOT NULL,
-    value bytea  NULL,
-    version bigint  NOT NULL,
+    store_id character varying(120) NOT NULL CHECK (store_id <> ''),
+    key character varying(120) NOT NULL,
+    value bytea NULL,
+    version bigint NOT NULL,
     PRIMARY KEY (store_id, key)
-);
-
+);
```

Comment:
Add newline.

Reply:
I earlier got comment about removing newlines/blanks at end of files, 
bit confused, let me know if i misunderstood.

Reply:
Sorry, please disregard. I thought that Github was showing the "No newline at end of file" symbol, but I guess I was mistaken.

Reply:
Ok, i might have fixed it, 
So just to clarify there "should" be a newline at EOF? 

Addressed other comments as well.

Reply:
Correct, it should contain a newline, just not an empty line.